recipes/cobra-meta/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cobra-meta" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/linxingchen/cobra/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 26f3cab513f8bb78d0aeb5ccfa3aed57bafb62f42bee1ac814e74e43c330a390
+
+build:
+  entry_points:
+    - cobra-meta = cobra:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('cobra-meta', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - biopython
+    - pysam
+    - numpy
+    - blast >=2.14.0,<3.0.0
+
+test:
+  commands:
+    - cobra-meta -v
+
+about:
+  home: https://github.com/linxingchen/cobra
+  license: MIT
+  summary: 'COBRA is a tool to get higher quality viral genomes assembled from metagenomes.'
+
+extra:
+  identifiers:
+    - "https://doi.org/10.1101/2023.05.30.542503"
```

```suggestion
    - doi:10.1101/2023.05.30.542503
```
You can list citations using the `doi:` tag.